pico_project/constants.json

```diff
@@ -0,0 +1,80 @@
+{
+    "features_list": {
```

Comment:
It'd probably be better to make `features_list` and `stdlib_examples_list` be a dict-of-dicts, rather than a dict-of-lists? (and then you could get rid of the `LibInfo` class?)

Also, given that these are a dict-of-lists anyway, the `_list` part of the name probably doesn't make much sense? :shrug:

Reply:
I'd prefer not to be making any major code changes to get this merged, since I'd be the one who has to do them.

Reply:
Fair enough!